### ISSUE TITLE
Make parsing ROS version configurable

### ISSUE THREAD

**Issue by PhilippeMorier** [Greenroom-Robotics/ros-typescript-generator#2]
First, thanks for your work :bouquet:!

**I'm submitting a ...**
  - [x] feature request
  - [x] PR #3

**Summary**

Currently, the parsing of ROS2 is hard-coded:
```ts
const messageDefinitions = parse(rosDefinition, { ros2: true });
```
See: https://github.com/Greenroom-Robotics/ros-typescript-generator/blob/master/src/lib/generateFromRosMsg.ts#L11

It would be awesome if it would be possible to change this in the `ros-ts-generator-config.json`?

**Background**

http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/CameraInfo.html can't be parsed because of e.g. `float64[9]  K # 3x3 row-major matrix`.

**Comment by MrBlenny** [comment]
Looks good! Thanks @PhilippeMorier 
I've released this in version `1.6.0`. You should get it straight away if you've been running it via npx.

One thing to note, I've changed it so `rosVersion` is in the ros-ts-generator-config.json. eg)
```js
{
  "output": "./generated/ros_msgs.ts",
  "rosVersion": 1, // 1 or 2
  "input": ....
}
```

**Comment by PhilippeMorier** [comment]
Amazing! Such a fast reply :rocket:!

Thanks a lot :bouquet:!
